enterprise/tailnet/pgcoord.go

```diff
@@ -168,8 +165,9 @@ func newPGCoordInternal(
 	}
 	go func() {
 		// when the main context is canceled, or the coordinator closed, the binder, tunneler, and
-		// handshaker always eventually stop.  Once they stop it's safe to cancel the querier context, which
-		// has the effect of deleting the coordinator from the database and ceasing heartbeats.
+		// handshaker always eventually stop. When the
+		// binder stops it updates all the peers handled
+		// by this coordinator to LOST.
```

Comment:
this is a bit misleading in this context, because the `handleBindings()` goroutine is not currently tracked by the waitGroup.

I'm also not sure that we want/need for the querier to wait until these workers are done before terminating.  Before it was because the querier(heartbeats) deleted the Coordinator row, which broke some of binder/tunneler queries.